notebook/notebook/templates/workshop/index.html

```diff
@@ -22,5 +15,8 @@ <h1>Welcome!</h1>
     <p>The notebook service is not currently available outside of
       workshops like this, but we (the Hail team) are working hard to
       change that.</p>
+    <p style="margin: 30px auto;display: flex;width: 125px;justify-content: space-between;align-items: center;">
```

Comment:
Should this go in the css?

Reply:
It certainly could. One off page, seemed simple, but I didn't want to put it in main.css since that is now global, and notebook.css also seemed inappropriate.